wordpress-mobile/WordPress-Android
# Issue/13268 remote site

Partially fixes #13268 

Merge instructions
1. Make sure https://github.com/wordpress-mobile/WordPress-Android/pull/13440 is merged
2. Remove "Not ready for merge" label
3. Merge this PR 

Small PR in which the "remote site id" is passed to the ActivityLogTypeFilterViewModel. Since the upcoming PR is related just to the UI itself, I thought it'd be better to create a separate PR.

To test:
There is nothing to test yet. If you really want to test it, you can put a breakpoint [here](https://github.com/wordpress-mobile/WordPress-Android/blob/696d9cf12abd947ea9c2794707dafead2ccd1067/WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterViewModel.kt#L39) and check the the id is not null. However, we'll test this before we remove the feature flag so it's not necessary.

PR submission checklist:
